contrib/tests/actions/chains/test_quickstart_key.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I generally recommend double quoting all the string values (as in key values) in YAML for safety.